### PR TITLE
andriy.safronov

### PR DESCRIPTION
**Pull request by AndSaf1**

### CODE REVIEW

File: andriy.safronov/05.rb [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+    (2..@max).each do |num|
+      array.push(num) if (1..num).select { |d| (num % d).zero? } == [1, num]
+    end
+    Array.new(array.size) do
+      puts array.join(', ')
+      array.push(array.shift)
+    end
+  end
+end
+MyClass.new(10).numbers
```
Comment (hawkprovvv) [inline andriy.safronov/05.rb:17]: Please, add an [empty line](https://thoughtbot.com/blog/no-newline-at-end-of-file) at the end of file.

File: andriy.safronov/05.rb [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+    array = []
+    (2..@max).each do |num|
+      array.push(num) if (1..num).select { |d| (num % d).zero? } == [1, num]
+    end
+    Array.new(array.size) do
+      puts array.join(', ')
+      array.push(array.shift)
+    end
+  end
+end
```
Comment (hawkprovvv) [inline andriy.safronov/05.rb:16]: Add an empty line after class

**Review by hawkprovvv — APPROVED** [review]
good work

File: andriy.safronov/05.rb (hunk 1 above)
Comment (hawkprovvv) [reply andriy.safronov/05.rb:17]: @AndSaf1

Comment (AndSaf1) [reply andriy.safronov/05.rb:17]: изображение
File without a new line, looks like the picture

Comment (hawkprovvv) [reply andriy.safronov/05.rb:17]: @AndSaf1 ok, I hadn't seen 🙂

File: andriy.safronov/06.rb [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+  return tor unless tor.is_a? Integer
+
+  k = tor.to_s.split('').map do |x|
+  if x.to_i.odd?
+    '-' + x + '-'
+  else
+    x
+  end
+  end.join.gsub('--', '-')
+  k.slice!(0) if k[0]=='-'
```
Comment (hawkprovvv) [inline andriy.safronov/06.rb]: @AndSaf1 please, resolve annotations from rubocop

**Pull request merged**